<launch>

	<!-- you can load an exisiting tree with <node ... args="tree.bt"> !-->
  <node pkg="liosam_2gridmap" type="liosam_2gridmap_node" name="convertor_node" output="screen">
    <param name="sub_topic" value="/lio_sam/mapping/slam_info"/>
	<param name="pub_map" value="/liosam_2gridmap/map"/>
	<param name="frame_id" value="map" />
	<param name="resolution" value="0.3"/>
	<param name="sensor_model/max_range" value="200.0" />
	<!-- filter_ground -->
	<param name="filter_ground" value="true" />
    <param name="ground_filter/plane_distance" value="2" />
	<param name="ground_filter/distance" value="2" />
	
  </node>

</launch>
